llvm/lib/Target/AArch64/AArch64Features.td

```diff
@@ -216,8 +211,10 @@ def FeaturePAN_RWV : SubtargetFeature<
 def FeaturePsUAO : SubtargetFeature< "uaops", "HasPsUAO", "true",
     "Enable v8.2 UAO PState (FEAT_UAO)">;
 
-def FeatureCCPP : SubtargetFeature<"ccpp", "HasCCPP",
-    "true", "Enable v8.2 data Cache Clean to Point of Persistence (FEAT_DPB)" >;
+let ArchExtKindSpelling = "AEK_DPB", MArchName = "dpb" in
```

Comment:
Please don't override the `ArchExtKindSpelling` for new extensions, let them default to e.g. `AEK_CCPP` so they correspond to the SubtargetFeature field name.

Reply:
I disagree with keeping AEK_CCPP. It is a made up name that doesn't match the FEAT_DPB from the architecture manual. I prefer overriding the `ArchExtKindSpelling` in the short term than having to rename all its occurences in the codebase. At least in this patch.